roles-in-tech/developer-advocate.md

```diff
@@ -149,14 +152,17 @@ Here is a link to different articles which will let you know much about a Develo
 - ["Reference8"](https://devrelweekly.com/issues)
```

Comment:
Please title reference links with title of reference page, and the kind of reference: article, blog, video, etc

Reply:
ohk sure. I will be doing that.